### PR TITLE
Bugfix/48

### LINKED ISSUE #48: NullObject issues
The below is throwing off the tests, and ultimately skewing the true comparisons. 

https://github.com/dimdenGD/ultimate-express/blob/3398194b8af64c2c9ade3a20bcc99c9b7804c6af/tests/index.js#L59

Whilst doing some testing in a real prod app, I noticed some behavior which was not the same as express. 

Pulling request headers, for example, produced "NullObject <[Object: null prototype] {}> {}"

Is there a reason why the type is being used, instead of just an empty object?

### PR DESCRIPTION
**Pull request by Pho3niX90**
closes #48

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
     "statuses": "^2.0.1",
     "tseep": "^1.2.2",
     "type-is": "^1.6.18",
-    "uWebSockets.js": "github:uNetworking/uWebSockets.js#v20.49.0",
+    "uWebSockets.js": "github:uNetworking/uWebSockets.js#v20.70.0",
```
Comment (dimdenGD) [inline package.json:62]: There's no 20.70.0 uWS.js, you're confusing it with C++ uWS

Comment (Pho3niX90) [reply package.json]: you are right. I was looking at the C++ version.

**Comment by Pho3niX90** [comment]
@dimdenGD I am finished with this now if you want to review. 
I have added a benchmark workflow as well, super basic, and can post to the PR, I think you just need to enable it in your settings. Would give a nice indication if a PR is slowing the code down etc.

**Comment by dimdenGD** [comment]
Why do {...new NullObject()}? It's like creating a fast object and then creating a slow one and putting it inside. Would be faster to just have {} at this point

**Comment by Pho3niX90** [comment]
> Why do {...new NullObject()}? It's like creating a fast object and then creating a slow one and putting it inside. Would be faster to just have {} at this point

My initial thoughts. But ran some benchmarks, and seems it's faster like this. See if you get the same results from it?

**Comment by dimdenGD** [comment]
I don't see how it's possible for it to be faster that way. Microbenchmarks are often deceptive

**Comment by Pho3niX90** [comment]
So these are the results (tests/sample/object_creation.js):

image

**Comment by dimdenGD** [comment]
You should probably test some reads+writes too. Also there's no point in using these spreads for internal stuff only used by library, only for public things like req.query and req.params

**Pull request merged**